app/config/graphql.js

```diff
@@ -0,0 +1,15 @@
+import convict from 'convict'
+
+const isTestOrDev = process.env.ENVIRONMENT_CODE === 'test' || process.env.ENVIRONMENT_CODE === 'dev'
```

Comment:
See if you can refactor this line to make it more concise.

If its more concise then its probably readable enough to use it directly on line 9 as the default value without declaring it as `isTestOrDev`